analyses/cms-open-data-ttbar/utils/file_input.py

```diff
@@ -158,4 +158,4 @@ def get_data_rootfiles_uri(self, query, as_signed_url=True, title="Untitled"):
             # update files for each process
             files_per_process.update({process: all_files[parent_key[self.filelist[:,1]==process]]})
 
-        return files_per_process
+        return files_per_process
```

Comment:
Same here as for HEPData: let's not change the newline to minimize files being touched.